src/transport/GroupPeerMessageCounter.cpp

```diff
@@ -172,6 +172,29 @@ CHIP_ERROR GroupPeerTable::RemovePeer(FabricIndex fabricIndex, NodeId nodeId, bo
     return err;
 }
 
+CHIP_ERROR GroupPeerTable::RemoveFabricIndex(FabricIndex fabricIndex)
```

Comment:
Again, this should probably be called `FabricRemoved`.

Reply:
Renamed